src/applications/mhv-medical-records/components/LabsAndTests/PathologyDetails.jsx

```diff
@@ -117,17 +117,23 @@ ${record.results} \n`;
         <h3 className="vads-u-font-size--base vads-u-font-family--sans">
           Site or sample tested
         </h3>
-        <p data-testid="pathology-sample-tested">{record.sampleTested}</p>
+        <p data-testid="pathology-sample-tested" data-dd-privacy="mask">
+          {record.sampleTested}
+        </p>
         <h3 className="vads-u-font-size--base vads-u-font-family--sans">
           Location
         </h3>
-        <p data-testid="pathology-location">{record.labLocation}</p>
+        <p data-testid="pathology-location" data-dd-privacy="mask">
+          {record.labLocation}
+        </p>
         <h3 className="vads-u-font-size--base vads-u-font-family--sans">
           Date completed
         </h3>
-        <p data-testid="date-completed">{record.date}</p>
+        <p data-testid="date-completed" data-dd-privacy="mask">
+          {record.date}
+        </p>
       </div>
-      <div className="test-results-container">
+      <div className="test-results-container" data-dd-privacy="mask">
         <h2>Results</h2>
         <InfoAlert fullState={fullState} />
         <p data-testid="pathology-report" className="monospace">
```

Comment:
Instead of the outer `div`, move the mask onto the `p` tag to keep the mask as tight as possible.

Reply:
fixed